planning/trajectory_optimization/gcs_trajectory_optimization.cc

```diff
@@ -647,9 +647,21 @@ void GcsTrajectoryOptimization::AddVelocityBounds(
 }
 
 std::pair<CompositeTrajectory<double>, solvers::MathematicalProgramResult>
-GcsTrajectoryOptimization::SolvePath(const Subgraph& source,
-                                     const Subgraph& target,
-                                     const GraphOfConvexSetsOptions& options) {
+GcsTrajectoryOptimization::SolvePath(
+    const Subgraph& source, const Subgraph& target,
+    const GraphOfConvexSetsOptions& specified_options) {
+  // Fill in default options.
+  GraphOfConvexSetsOptions options = specified_options;
+  if (!options.convex_relaxation) {
+    options.convex_relaxation = true;
+  }
+  if (!options.preprocessing) {
+    options.preprocessing = true;
```

Comment:
Maybe `preprocessing` could be `false` by default? It can be expensive if the graph has many edges (should be an LP per edge). And it seems the kind of thing that the expert user wants to activate to have a performance boost for some particular problems?